governance/polls/Proposal - Rates V2 Poll - October 12, 2020.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Rates V2 Poll - October 12, 2020
+summary: Signal your support to execute the Rates V2 plan.
+discussion_link: https://forum.makerdao.com/t/signal-request-rates-v2/4481
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Rates V2 Poll - October 12, 2020
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) which gives Maker Governance the chance to signal their support for the implementation of Rates V2 plan.
```

Comment:
... of **the** Rates V2 plan

Reply:
Also: The MakerDAO Governance Facilitator has placed a governance poll into the voting system on behalf of the Maker Governance community which...